runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -232,4 +232,15 @@ default int infoBoxSize()
 	{
 		return 35;
 	}
+
+	@ConfigItem(
```

Comment:
This should probably be somewhere under gameAlwaysOnTop, at bottom there are infobox-related settings